assets/airports/mkjp.json

```diff
@@ -177,19 +208,184 @@
+            "body": [""],
+            "exitPoints": {
+                "OMPAL": ["OMPAL"]
+            },
+            "draw": [
+                ["GUDIL", "OMPAL*"]
+            ]
+        },
+        "RADOK7": {
+            "icao": "RADOK7",
+            "name": "Radok Seven",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["ELSER"]
+            },
+            "body": ["RADOK"],
+            "exitPoints": {
+                "RADOK": ["RADOK"]
+            },
+            "draw": [
+                ["RADOK*"]
+            ]
+        },
+        "TIBEL3": {
+            "icao": "TIBEL3",
+            "name": "Tibel Three",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["GUDIL"]
+            },
+            "body": [""],
+            "exitPoints": {
+                "TIBEL": ["TIBEL"]
+            },
+            "draw": [
+                ["GUDIL", "TIBEL*"]
+            ]
+        },
+        "URMAN3": {
+            "icao": "URMAN3",
+            "name": "Urman Three",
+            "rwy": {
+                "MKJP12": [""],
+                "MKJP30": ["GUDIL"]
+            },
+            "body": [""],
+            "exitPoints": {
+                "URMAN": ["URMAN"]
+            },
+            "draw": [
+                ["GUDIL", "URMAN*"]
+            ]
         }
     },
     "stars": {
         "ELSER5": {
             "icao": "ELSER5",
             "name": "Elsers Five",
             "entryPoints": {
-                "DATOM" : ["DATOM", "R205Q", "MLY17DA", "MLY17DB", "MLY17DC", "MLY17DD", "MLY17DE", "MLY17DF", "MLY17DG", "MLY17DH", "MLY17DI", "MLY17DI", "MLY17DJ", "MLY17DK", "MLY17DL", "MLY17DM", "MLY17DN", "MLY17DO", "MLY17DP", "MLY17DQ", "MLY17DR", "R290Q", "MLY17KAA", "R300Q", "ELSER"],
-                "SAVEM"  :["SAVEM", "R343T", "HAVBO", "MLY17SA", "MLY17SB", "MLY17SC", "MLY17SD",  "R318Q", "ZINAN", "MLY17LA", "MLY17LB", "R300Q", "ELSER"],
-                "RADOK"  :["RADOK", "R355T", "R355Q", "R343Q", "HAVBO", "MLY17SA", "MLY17SB", "MLY17SC", "MLY17SD", "R318Q", "ZINAN", "MLY17LA", "MLY17LB", "R300Q", "ELSER"],
-                "KESPA" : ["KESPA", "R290Q", "MLY17KAA", "R300Q", "ELSER"],
-                "LIBEX":  ["LIBEX", "R318S", "R318Q", "ZINAN", "MLY17LA", "MLY17LB", "R300Q", "ELSER"],
-                "ALPEN":  ["ALPEN", "R173Q", "ACEDA", "MLY17AB", "MLY17AC", "MLY17AD", "MLY17AE", "MLY17AF", "R205Q", "MLY17DA", "MLY17DB", "MLY17DC", "MLY17DD", "MLY17DE", "MLY17DF", "MLY17DG", "MLY17DH", "MLY17DI", "MLY17DI", "MLY17DJ", "MLY17DK", "MLY17DL", "MLY17DM", "MLY17DN", "MLY17DO", "MLY17DP", "MLY17DQ", "MLY17DR", "R290Q", "MLY17KAA", "R300Q", "ELSER" ],
+                "DATOM" : ["DATOM", "R205Q", "_MLY17DA", "_MLY17DB", "_MLY17DC", "_MLY17DD", "_MLY17DE", "_MLY17DF", "_MLY17DG", "_MLY17DH", "_MLY17DI", "_MLY17DI", "_MLY17DJ", "_MLY17DK", "_MLY17DL", "_MLY17DM", "_MLY17DN", "_MLY17DO", "_MLY17DP", "_MLY17DQ", "_MLY17DR", "R290Q", "_MLY17KA", "R300Q", "ELSER"],
```

Comment:
R205Q should be a fly-over fix. It should also be renamed to _MLY205019, so it would look like this
`"^_MLY205019"`